### ISSUE TITLE
Broken links in 'Install on Linux' page

### ISSUE THREAD

**Issue by obydog002** [FAForever/downlords-faf-client#3133]
[<img alt="Issuehunt badges" src="https://img.shields.io/badge/IssueHunt-%245%20Funded-%2300A156.svg" />](https://oss.issuehunt.io/r/FAForever/downlords-faf-client/issues/3133)

**Describe the bug**

[Install on Linux](https://github.com/FAForever/downlords-faf-client/wiki/Install-on-Linux)

The guide link is broken and the Video link could be updated to the newer version.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $5.00)

- [<img src='https://avatars.githubusercontent.com/u/42963271?v=4' alt='fox2code' width=24 height=24> fox2code](https://oss.issuehunt.io/u/fox2code) ($5.00)

### Submitted pull Requests
- [#3419 Fix Linux install documentation links (#3133)](https://oss.issuehunt.io/r/FAForever/downlords-faf-client/pull/3419)
---

### Tips

- Checkout the [Issuehunt explorer](https://oss.issuehunt.io/r/FAForever/downlords-faf-client/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://oss.issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

**Comment by tatsujb** [comment]
yeah I always update the description of the video to the new one.

the new one is this one: https://www.youtube.com/watch?v=4Wm-yZpV_q

**Comment by obydog002** [comment]
that is broken link

**Comment by sulikdan** [comment]
@obydog002 I suppose it should be this oen - https://www.youtube.com/watch?v=4Wm-yZpV_q8 (just checked his youtube)

**Comment by shivam-gujar** [comment]
Hi! I'd like to work on this issue. Please assign it to me.

**Comment by AmanRawat33** [comment]
Hi is this issue closed ? If not I would like to work on it. Please assign it to me I'll close it in 2 days.

**Comment by rishi-jat** [comment]
@obydog002 I would like to work on this issue is it still open ?
